lib/ansible/modules/async_wrapper.py

```diff
@@ -293,8 +293,8 @@ def main():
                     continue
 
             notice("Return async_wrapper task started.")
-            end({"failed": 0, "started": 1, "finished": 0, "ansible_job_id": jid, "results_file": job_path,
-                 "_ansible_suppress_tmpdir_delete": (not preserve_tmp)}, 0)
+            end({"failed": False, "started": True, "finished": False, "ansible_job_id": jid, "results_file": job_path,
+                 "_ansible_suppress_tmpdir_delete": (not preserve_tmp)}, False)
```

Comment:
The last argument is an exit code, so it shouldn't be changed:

```suggestion
                 "_ansible_suppress_tmpdir_delete": (not preserve_tmp)}, 0)
```